test/integrationtests/LorisIntegrationTest.class.inc

```diff
@@ -36,10 +32,12 @@ abstract class LorisIntegrationTest extends PHPUnit_Framework_TestCase
                 'Active' => 'Y',
                 'Examiner' => 'N',
                 'Password_md5' => 'a601e42ba82bb37a68ca3c8b7752f2e222',
+                'Password_hash' => null,
                 'Password_expiry' => '2099-12-31',
-                'Pending_approval' => false
+                'Pending_approval' => 'N'
             )
         );
+
         $this->DB->run(
```

Comment:
because DB->insert doesn't support a subselect as the insert value, and it's trying to give all permissions to the user